src/pkg/podman/podman.go

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2019 – 2022 Red Hat Inc.
+ * Copyright © 2019-2022 Red Hat Inc.
```

Comment:
Stray unrelated change.

Reply:
This change was made by mistake after fixing some problems I had.

Reply:
Unresolving, because it's still there.  :)

Reply:
Modified :)